.gitignore

```diff
@@ -1 +1,3 @@
 node_modules
+.DS_Store
+.idea/
```

Comment:
this should probably be in your own global gitignore

Reply:
.DS_Store is very common.
Also is webstorm.